### DISCUSSION TITLE
Is it possible to generate two related patients one is mother and one is her child?

### DISCUSSION THREAD

**Discussion by mduna** [synthetichealth/synthea#1024] [Q&A] [answered]
I like to simulate a case between mother and child. Is it possible to create a module for it?  Thanks.

**Comment by jawalonoski** [comment, marked as answer]
While women is Synthea can get pregnant and give birth, there is no link to the children, nor are the children explicitly generated and simulated.

You could create a module where activity occurs relating to what you want to simulate, and only add details to the mother's health record where it would show up. Or vice versa... for example, simulate a child and determine if there is a "family history of X" or birth complication.

But there is no explicit mother/child relationship that you can programmatically access.

**Reply by mduna** [reply]
Thank you for the suggestion, that helps.
